tests/PKSim.Tests/Presentation/PopulationSettingsDTOSpecs.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using OSPSuite.BDDHelper;
+using OSPSuite.Utility.Validation;
+using PKSim.Presentation.DTO.Populations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PKSim.Presentation
+{
+   public class concern_for_PopulationSettingsDTO : ContextSpecification<PopulationSettingsDTO>
+   {
+      protected override void Context()
+      {
+         sut = new PopulationSettingsDTO();
+         sut.NumberOfIndividuals = 5;
+         sut.ProportionOfFemales = 50;
+      }
+
+      [TestCase(50, true)]
+      [TestCase(101, false)]
+      [TestCase(-1, false)]
+      public void FemaleRate_should_be_properly_bounded(int femaleRate, bool expectation)
+      {
+         sut.ProportionOfFemales = femaleRate;
+         Assert.AreEqual(expectation, sut.IsValid());
+      }
```

Comment:
Sorry I meant sut.IsVali().shouldbbeequaldTO(expectation)

We just almost never used Assert.AreEqual. But this is fine for me as well

Reply:
@msevestre I see, no you are right... I am changing it ;)